upstream-community-operators/portworx/portworxoperator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,475 @@
+    mediatype: image/png
+  description: |
+    Portworx Enterprise is the most widely-used and reliable cloud-native
+    storage solution for production workloads and provides high-availability,
+    data protection and security for containerized applications.
+
+
+    Portworx Enterprise enables you to migrate entire applications,
+    including data, between clusters in a single data center or cloud, or between
+    clouds, with a single kubectl command.
+
+
+    The cloud native storage and data management platform that enterprises trust
+    to manage data in containers now has an operator which simplifies the
+    install, configuration, upgrades and manages the Portworx Enterprise cluster
+    lifecycle.
+
+    ## Prerequisite
+
+    Portworx runs as a privileged pod. Hence you need to add the Portworx service accounts to the privileged security context.
+
+    ```
+    oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:px-lh-account
```

Comment:
Since we have your Operator already in the community category of OpenShift, you might want to reconsider adjusting this description to community of OperatorHub.io, which is the wider Kubernetes audience.

Reply:
@piyush-nimbalkar any update here?

Reply:
@dmesser I have updated the review with the changed description. Have added some more validations in the CRD and changed the version from 1.0.0 to 1.0.3.